vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -71,14 +66,34 @@ mkdir bundle
 ```
 2. Run the bundler:
 ``` sh
-ent bundler from-env --location bundle --code YOUR-TEST-BUNDLE --description “Your Test Bundle”
+ent bundler from-env --location bundle --code YOUR-BUNDLE-NAME --description “Your Exported Bundle”
 ```
-The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files and assemble the top-level descriptor file.
+The bundler will inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files, and assemble the top-level descriptor file.
 
 ``` 
 $ ls bundle
 assets      contentModels  contents         fragments  labels     pageModels  resources
 categories  contentTypes   descriptor.yaml  groups     languages  pages       widgets
 ```
 
-You now have a complete Entando project structure! You can inspect the output to edit the exported components or deploy it to another Entando Application.
+You now have a complete Entando project structure for a git-based bundle! You can inspect the output to edit the exported components or [deploy](publish-project-bundle.md) it to another Entando Application. 
+::: tip Note:
+To convert this project to a docker-based bundle, continue with the steps below. 
+:::
+
+### Construct the Docker-based Bundle Structure
+
+1. Initialize a new bundle and switch to that directory:
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+cd YOUR-BUNDLE-NAME
+```
+2. Copy the resources from the bundle directory, with the exception of microservices (/plugins) and micro frontends (/widgets), to the `platform` directory:
```

Comment:
I'm on the fence here. I provided a more general command but describing the process might actually be good enough. One nuance is that the widgets can contain non-MFE widgets, like in our basic tutorial. We probably need to clarify that the /plugins and /widgets will need to be handled manually because of things like that. You have a good start on that below.

Reply:
@Lyd1aCla1r3, skip this section for now, lines 84 to the end, it is going to get reworked

Reply:
is the file path used for the cp set in stone?